h2o-r/h2o-package/R/frame.R

```diff
@@ -4931,3 +4936,130 @@ h2o.target_encode_apply <- function(data, x, y, target_encode_map, holdout_type,
   
   return(te_frame)
 }
+
+.newExprMap <- function(op,...) .newExprListMap(op,list(...))
+
+.newExprListMap <- function(op,li) {
+    node <- structure(new.env(parent = emptyenv()))
+    .set(node,"op",op)
+    .set(node,"eval",li)
+    node
+}
+
+.arrayArgumentHelper <- function(arr) {
+    if(length(arr) == 1) {
+        arr <- .quote(arr)
+    } else {
+        arr <- unlist(arr)
+    }
+    return(arr)
+}
+
+#'
```

Comment:
Is there a strong reason to include TE in frame.r. I think it deserves its own file (just like in python).

Reply:
@michalkurka No, no strong reasons at all. Just followed a trend... previous version of R's TE was(is) in this file. I will move it, thanks